sgkit/io/vcf/vcf_reader.py

```diff
@@ -225,6 +241,17 @@ def vcf_to_zarrs(
         Width (number of samples) to use when storing chunks in output, by default 1,000.
     output_storage_options
         Any additional parameters for the storage backend, for the output (see ``fsspec.open``).
+    ploidy : int
+        The (maximum) ploidy of genotypes in the VCF file.
+    mixed_ploidy : bool
+        If true, genotype calls with fewer alleles than the specified ploidy will be padded
```

Comment:
Nit: we've been using True to refer to the python value (not in monospaced font though). Not sure if there's a best practice to follow for that.